### DISCUSSION TITLE
Add option to always open all applications maximized

### DISCUSSION THREAD

**Discussion by yglodt** [rxhanson/Rectangle#800] [Ideas]
Hello,

thank you for Rectangle which is a great app which I use from day 1 since I use a Mac.

I find myself aways maximizing windows by pressing Option-Ctrl-Enter, since some apps seem to to remember their last window size.

I wonder if you could add an option "Always open all apps maximized" (or "Automatically maximize all windows on launch") so that the manual maximizing would not be necessary anymore.

Best regards

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have the bandwidth to add this in at the moment, but see no reason to turn down a good pull request with this functionality. If anyone wants to give it a go, I would prefer that it starts out as a terminal command configuration and is not a checkbox in the UI, until it's determined that a lot of people desire this functionality.
